Makefile

```diff
@@ -36,6 +36,8 @@ prereq: make-requirements.txt
 	pyenv install --skip-existing $(PY37)
 #	pyenv global system $(PY34) $(PY35) $(PY36) $(PY37)
 	pyenv global system $(PY35) $(PY36) $(PY37)
+	@# Ensure all Python versions are registered for this project
+	@eval "$$(pyenv init -)" && pyenv versions --bare > .python-version;
```

Comment:
As discussed makes sense to just use the already present `make-requirements.txt` with something like `awk -F'=' '{print $2}' make-requirements.txt >.python-version`